src/containers/OrderUpdate.vue

```diff
@@ -75,6 +80,90 @@ export default {
       } catch (err) {
         this.handleError(err);
       }
+    },
+
+    onPrint() {
+      if (!this.params) return [];
+
+      const { categories, squares, parts } = this.params;
+
+      var dd = {
+        content: [
+          {
+            stack: [
+              this.$t("invoice"),
+              {
+                text: `${dayjs(this.order.date).format("DD.MM.YYYY")}`,
```

Comment:
just dayjs(this.order.date).format("DD.MM.YYYY")

Reply:
done